nvflare/app_common/job_launcher/server_process_launcher.py

```diff
@@ -21,27 +21,29 @@
 
 
 class ServerProcessJobLauncher(ProcessJobLauncher):
-    def get_command(self, launch_data, fl_ctx) -> (str, dict):
+    def get_command(self, job_meta, fl_ctx) -> (str, dict):
         new_env = os.environ.copy()
+        workspace_obj: Workspace = fl_ctx.get_prop(FLContextKey.WORKSPACE_OBJECT)
+        job_id = job_meta.get(JobConstants.JOB_ID)
+        app_custom_folder = workspace_obj.get_app_custom_dir(job_id)
+        if app_custom_folder != "":
+            add_custom_dir_to_path(app_custom_folder, new_env)
 
+        command = self.generate_server_command(job_meta, fl_ctx)
+        return command, new_env
+
+    def generate_server_command(self, job_meta, fl_ctx):
```

Comment:
if this is moved out as regular common method, then ServerDockerJobLauncher does not need to inherit ServerProcessJobLauncher?

Reply:
same as the client job launcher.